param/morse.yaml

```diff
@@ -45,3 +45,5 @@ sim:
     # bottom_camera_resolution: [1280, 960, 60]
 
     create_teleport_actuator: false
+
```

Comment:
Can we get the same comment as on line 37 in morse-prototype.yaml here?

Reply:
fixed in 7af24a5